### PR TITLE
Added chefspec tests for mon recipe and updated rubocop standards

### PR DESCRIPTION
**Pull request by cthalmann**
This PR adds chefspec tests for the mon.rb recipe. Also, while writing tests, I noticed that the rubocop standards had not changed (said the maximum characters for a line was 80 instead of 120) and so I updated the .rubocop.yml file. As a result, I had to make other rubocop changes to the cookbook. 

How to test chefspec tests:
git checkout cthalmann/test-mon
rspec spec/mon_spec.rb

### CODE REVIEW

File: spec/mon_spec.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      end
+      it do
+        expect(chef_run).to create_template("#{chef_run.node['munin']['basedir']}/plugin-conf.d/haproxy").with(
+          source: 'munin/haproxy.erb',
+          owner: 'root',
+          group: 'root',
+          mode: 0644
+        )
+      end
+    end
```
Comment (fahlmant) [inline spec/mon_spec.rb]: What about the `munin_plugin` resource on line 28 of the mon recipe?

Comment (ramereth) [inline spec/mon_spec.rb]: @fahlmant ``munin_plugin`` is a definition and can't be tested like we normally do. The best we can do is test for the resources it creates (at least a few of the important ones).

Comment (fahlmant) [inline spec/mon_spec.rb]: +1 to that, as long as we have some kind of indication that it ran

Comment (ramereth) [inline spec/mon_spec.rb]: I would be fine skipping testing the ``munin_plugin`` definition for now since its so difficult to do.

**Comment by cthalmann** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v2.3.2.  Have a nice day!
